units/units.cpp

```diff
@@ -1710,6 +1738,55 @@ static double getPrefixMultiplier(char p)
     }
 }
 
+/// Generate the prefix multiplier for SI units
+static double getStrictSIPrefixMultiplier(char p)
+{
+    switch (p) {
+        case 'm':
+            return 0.001;
+        case 'k':
+        case 'K':
```

Comment:
Is this actually SI?

Reply:
removed the capital K from the strict SI for kilo.